src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,150 @@
+package duke;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeParseException;
+import java.util.Scanner;
+
+/**
+ * The {@code Storage} class. Deals with writing to, and reading from, text files.
+ */
+public class Storage {
+    private final TaskList tasks;
```

Comment:
Same comment as previous, about naming this variable considering that its type is `TaskList`